torch/_inductor/compile_fx.py

```diff
@@ -664,78 +625,15 @@ def codegen_and_compile(
                 # need to do any post-compilation steps: we just return the string,
                 # which is the filename of the compiled code.
                 return compiled_graph
```

Comment:
You can see above that post compile does nothing in AOT mode. I want to eventually have AOTInductor do its own custom post compile logic at this point. However, I think there really should only be one post compile method.